.github/candidate-specs.md

```diff
@@ -0,0 +1,10 @@
+---
+title: New specs for review
+assigneeds: tidoust, dontcallmedom
```

Comment:
I suspect s/assigneeds/assignees will fix the fact that the bot created an issue with assigning it to us.

Reply:
that was a feature, but oh well